tests/Diagnostics.HealthChecks.UnitTests/OmexHealthCheckPublisherHostedServiceTests.cs

```diff
@@ -494,12 +494,12 @@ public async Task RunAsync_CanFilterHealthChecks_RegistrationParameters()
 
 				for (int i = 0; i < publishers.Length; i++)
 				{
-					string[] entries = [.. publishers[i].Entries.SelectMany(e => e.report.Entries.Select(e2 => e2.Key)).OrderBy(k => k)];
+					IEnumerable<string> entries = publishers[i].Entries.SelectMany(e => e.report.Entries.Select(e2 => e2.Key)).OrderBy(k => k);
```

Comment:
Change is to fix flakey unit test (collection sometimes reports it's being modified) switching to IEnumerable.